tests/contract/knative-channel-kafka/OWNERS

```diff
@@ -0,0 +1,5 @@
+reviewers:
+  - serverless-reviewers
```

Comment:
since this test is eventing related and not serveless, I would suggest to choose `event-bus-approvers`.